README.md

```diff
@@ -1,3 +1,10 @@
+# ⚠️ Deprecation Notice ⚠️
+
+# **This Package is No Longer Maintained**
+
+This package is no longer being maintained. For the most up-to-date information and API documentation, please refer to the official Docsumo API [documentation](https://apidocs.docsumo.com/).
```

Comment:
We are maintaining it here https://support.docsumo.com/reference/getting-started-with-your-api
not on apidocs